diffkemp/diffkemp.py

```diff
@@ -29,6 +29,12 @@
 
 
 def build(args):
+    # If the symbol list is specified, check that the file exists.
+    if args.symbol_list is not None and not os.path.isfile(args.symbol_list):
+        sys.stderr.write(
+            "Error: specified symbol_list file does not exist\n"
```

Comment:
Nice catch! Could we use the same error message that `build_kernel` uses when the symbol list file doesn't exist (or is empty)?

Reply:
So I found out that neither of the build commands (`build-kernel`, `build`, `llvm-to-snapshot`) were checking if the specified `symbol_list` file exist, so I added the check to all commands. Also I added a check to the `build` command if it was possible to extract some symbols from the `symbol_list` / project itself - more info in the commit messages.